src/java/org/apache/cassandra/io/sstable/format/big/BigTableReader.java

```diff
@@ -374,4 +380,20 @@ public ClusteringPrefix<?> getLowerBoundPrefixFromCache(DecoratedKey partitionKe
         }
 
     }
-}
+
+    @Override
+    public IScrubber getScrubber(LifecycleTransaction transaction, OutputHandler outputHandler, IScrubber.Options options)
+    {
+        ColumnFamilyStore cfs = Schema.instance.getColumnFamilyStoreInstance(metadata());
```

Comment:
Could we do this without referencing the higher-level CFS and Schema concepts in the sstable reader implementation?

Perhaps implement it in the format and pass the CFS as an argument?

Reply:
I wanted to avoid requiring CFS as an argument but you are right.

Reply:
looks good